invenio_rdm_records/resources/resources.py

```diff
@@ -43,6 +44,22 @@ def p(route):
     #
     # Review request
     #
+    @request_search_args
+    @request_view_args
+    @response_handler(many=True)
+    def search_community_records(self):
```

Comment:
Can you please move this down at the very bottom with a similar comment on top?
the comment here says `# Review request` and it is not releated to that